IBC.md

```diff
@@ -0,0 +1,412 @@
+# IBC interfaces for CosmWasm contracts
+
+If you import `cosmwasm-std` with the `stargate` feature flag, it will expose a
+number of IBC-related functionality. This requires that the host chain is
+running an IBC-enabled version of `wasmd`, that is `v0.16.0` or higher. You will
```

Comment:
Do I understand correctly that other projects use the "x/wasm" module, and not the wasmd node application?


```suggestion
running an IBC-enabled version of `x/wasm`, that is `v0.16.0` or higher. You will
```